internal/pkg/cli/validate.go

```diff
@@ -85,3 +95,42 @@ func isCorrectFormat(s string) bool {
 	}
 	return valid
 }
+
+func basicPortValidation(val interface{}) error {
+
+	var err error
+
+	switch val := val.(type) {
+	case []byte:
```

Comment:
I wonder if we really need validation for "[]byte" and "string" format for the flag, because we already specify `Uint16Var` when defining the flag so it is supposed to always be "unit16" when getting passed in through a flag.

Reply:
It gets passed as a []byte when the user enters it with a prompt.